nimbus/src/enrollment.rs

```diff
@@ -110,7 +110,7 @@ impl ExperimentEnrollment {
         }
         let enrollment = Self {
             slug: experiment.slug.clone(),
-            status: EnrollmentStatus::new_enrolled(EnrolledReason::OptIn, branch_slug),
+            status: EnrollmentStatus::new_enrolled(EnrolledReason::OptIn, branch_slug, &experiment.feature_ids[0],)
```

Comment:
nit: slightly odd trailing `,` here - I think it should move out of the parens (although you could also argue it should be in *both* places, but IMO that would also be odd, so 🤷 )

Reply:
I couldn't find any documentation in this repo on what our policy / usage theory is for `cargo fmt` in this project.  Can someone describe the policy in a way that we could stuff into `README.md`?

Reply:
Nevermind -- I see that `cargo fmt` is in the CI!

Reply:
`cargo fmt` did the trick.